### ISSUE TITLE
where did simple pdf export go?

### ISSUE THREAD

**Issue by vitopepito** [shd101wyy/markdown-preview-enhanced#533]
in the previous version, pdf export was straightforward and simple. Now there are more options, but none of them really works for me. 
- open in browser + print + save as pdf: too many steps and produces large font size
- phantomjs: pdf export on mac is broken, the page content does not fit on the pdf page format, documented here https://github.com/ariya/phantomjs/issues/12685
- pandoc: works nicely but cannot produce github style pdfs but introduces its own template

**Comment by shd101wyy** [comment]
Yes it is removed. [#522](https://github.com/shd101wyy/markdown-preview-enhanced/issues/522).  
For your first option, if you think the font size is too large, then you can [Customize Css](https://shd101wyy.github.io/markdown-preview-enhanced/#/customize-css) to control the font size.  

```less
html body {
  @media print {
    font-size: 8px; // <= the font size
  }
}
```
